integration/src/test/resources/spring-test/object-factory-context.xml

```diff
@@ -46,4 +46,14 @@
         <property name="updateDescriptionService" ref="updateDescriptionService" />
         <property name="repositoryObjectLoader" ref="repositoryObjectLoader" />
     </bean>
+
+    <bean id="fileFactory" class="edu.unc.lib.boxc.integration.factories.FileFactory">
```

Comment:
Should also initialize another bean for workFactory

Reply:
would it have all the same content as the file factory bean?

Reply:
I think so, except that it should also receive a fileFactory as a property